### DISCUSSION TITLE
unbale to destroy the server after creating golden-ami from the same server

### DISCUSSION THREAD

**Discussion by shreebadiger** [learndevopsonline/learndevopsonline#1709] [Q&A] [answered]
![Screenshot 2023-12-30 093118](https://github.com/learndevopsonline/learndevopsonline/assets/145904607/afa90a4b-22f1-4b18-91b0-84258c869dd1)

**Comment by sreedharm07** [comment]
mention state file in command line 
same like how you mention state file for init

**Reply by shreebadiger** [reply]
Can you tell the steps

**Reply by sreedharm07** [reply]
@shreebadiger  can you join the lab session now

**Reply by sreedharm07** [reply]
@shreebadiger please join the lab session now.

**Reply by shreebadiger** [reply, marked as answer]
Ok

**Reply by shreebadiger** [reply]
Thanks issue resolved
